integration/sawtooth_integration/tests/test_scripts/local_transactor_permissioning.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
If this really, truly, absolutely cannot be put into the yaml file, then the test file should announce prominently that it expects this script to be run and what it does and the script should be named the same as the test file and yaml file (but with the `.sh` suffix).

I also don't think it should go into a subdirectory; `/test_scripts` suggests to me that these are scripts that maybe run for every test or something like that.

Reply:
I will do the renaming. I couldn't get it to work from within a bash -c command.